### PR TITLE
[Maintenance] Reorganize mobile components

### PR DESCRIPTION
**Pull request by joel-jeremy**
Moved mobile components under the `components/mobile` folder and moved exported components to separate files.

### CODE REVIEW

File: packages/desktop-client/src/components/mobile/accounts/Account.jsx [hunk 1]
```diff
@@ -138,43 +133,50 @@ export function Account(props) {
...
-      updateQuery(
-        queries.makeTransactionSearchQuery(
-          currentQuery,
-          searchText,
-          state.dateFormat,
-        ),
-      );
-    }
-  }, 150);
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/mobile/accounts/Account.jsx:158]: **❓ question:** ‏what arg are we missing in the deps here?

**Review by MatissJanis — COMMENTED** [review]
Overall LGTM. Just one clarification question.

File: packages/desktop-client/src/components/mobile/accounts/Account.jsx (hunk 1 above)
Comment (joel-jeremy) [reply packages/desktop-client/src/components/mobile/accounts/Account.jsx:158]: Reason for adding that is linter cannot identify proper dependencies of the function due to it being wrapped in `debounce`. Any ideas to get around this?
image

Comment (MatissJanis) [reply packages/desktop-client/src/components/mobile/accounts/Account.jsx:158]: I think easiest solution would be to install and use something like this: https://usehooks.com/usedebounce

But with this we're venturing a bit outside the scope of the PR.. so feel free to do with this comment as you please ;)

**Review by MatissJanis — APPROVED** [review]

**Review by MatissJanis — APPROVED** [review]

**Review by MatissJanis — APPROVED** [review]

**Pull request merged**